libs/viewer/src/SimpleViewer.cpp

```diff
@@ -600,9 +600,14 @@ void SimpleViewer::updateUserInterface() {
     if (ImGui::CollapsingHeader("View")) {
         ImGui::Indent();
 
-        bool dither = mSettings.view.dithering == Dithering::TEMPORAL;
-        ImGui::Checkbox("Dithering", &dither);
-        enableDithering(dither);
+        ImGui::Checkbox("Post-processing", &mSettings.view.postProcessingEnabled);
```

Comment:
Oh nice, this setting already existed so we should have exposed it anyway.  We might want to update the remote web page at some point.